task/06-conditions-n-loops-tasks.js

```diff
@@ -192,7 +223,12 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    if(a > b){
+        let temp = a;
+        a = b;
+        b = temp;
+    }
+    return (isStartIncluded ? '[' : '(') + a.toString() + ', ' + b.toString() + (isEndIncluded ? ']' : ')');
```

Comment:
Try to avoid these multi ternary operators.
It's very hard to read and maintain
if-else  much better